src/Stryker.CLI/Stryker.CLI/packages.lock.json

```diff
@@ -1181,12 +1160,12 @@
       "stryker": {
         "type": "Project",
         "dependencies": {
-          "Buildalyzer": "4.1.3",
+          "Buildalyzer": "4.1.4",
           "DotNet.Glob": "3.1.3",
           "FSharp.Compiler.Service": "38.0.0",
           "LibGit2Sharp": "0.27.0-preview-0182",
           "Microsoft.CodeAnalysis.CSharp": "4.1.0",
-          "Microsoft.Extensions.Configuration": "6.0.0",
+          "Microsoft.Extensions.Configuration": "6.0.1",
           "Microsoft.Extensions.Configuration.Json": "6.0.0",
```

Comment:
We will get these version updates automatically, you do not need to include them in this PR unless they are required for .net 6.

Reply:
I'll revert those then. 😄

Reply:
done